packages/docusaurus-mdx-loader/src/remark/relativePath/index.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+const visit = require('unist-util-visit');
+
+const plugin = () => {
+  const transformer = (root) => {
+    visit(root, 'image', (node) => {
+      if (node.url.startsWith('.')) {
+        node.type = 'jsx';
+        node.value = `<img ${node.alt ? `alt={"${node.alt}"}` : ''} ${
+          node.url ? `src={require("!url-loader!${node.url}").default}` : ''
```

Comment:
Forcing the url loader is to avoid the conflict with ideal image plugin right?

Reply:
Yes